src/Components/MovieTile/MovieTile.module.css

```diff
@@ -0,0 +1,48 @@
+.container {
+  width: 250px;
+  color: white;
+  background-color: transparent;
+  overflow: hidden;
+  font-weight: 100;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
+    'Droid Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container img {
+  width: 100%;
+  display: block;
+}
+
+.container > div {
```

Comment:
Same here.
It's better to define styles for some class here and to set this class to all `div`-s where these stykes are needed.